doc/4-instrument/README.md

```diff
@@ -126,14 +126,26 @@ finally, the sequence of values can be directly edited in the text box at the bo
 - in arpeggio macros, a value starting with a `@` is a fixed value as described above.
 - in bitmask-style macros, the values are added up in binary and converted to decimal. see [the hexadecimal guide](../1-intro/hex.md) for more info.
 
-
+in all cases except bitmask macros, right-clicking on the graph opens up a small editing menu:
+- **copy**: copy the entire macro.
+- **paste**: paste a copied macro. changes the macro length to match.

```

Comment:
"Fry a cut potato". Obviously a potato has to be cut before frying...

Replace with "paste the macro in the clipboard".